### PR TITLE
TDAP058-Code-Refactoring-And-Cleanup

### PR DESCRIPTION
**Pull request by amit-ib**
1. Prepared "Header" and "TodoAppContainer" as 2 new component
2. Cleaned up App.tsx

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -47,86 +26,12 @@ function App() {
...
-            <div>
-              <TodoForm
-                todos={state.tasks}
-                status={state.status}
-                setActiveId={setActiveId}
-                activeId={activeId}
-                setFilter={setFilter}
-                setLoading={setLoading}
-              />
+      <Header setIsLogedin={setIsLogedin} isLogedin={state.isLogedin} />
```
Comment (akash-dhande-ib) [inline src/App.tsx:31]: Do we need to send state to header, as we are using redux for state management

File: src/components/todo/TodoAppContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            )}
+        </div>
+      )}
+      {!state.isLogedin && (
+        <Login
+          isLogedin={state.isLogedin}
+          setLoading={setLoading}
+          setIsLogedin={setIsLogedin}
+        />
+      )}
```
Comment (akash-dhande-ib) [inline src/components/todo/TodoAppContainer.tsx:95]: I think login is not part of todo

File: src/App.tsx (hunk 1 above)
Comment (amit-ib) [reply src/App.tsx]: Updated

File: src/components/todo/TodoAppContainer.tsx (hunk 2 above)
Comment (amit-ib) [reply src/components/todo/TodoAppContainer.tsx]: Moved out and placed in App

File: src/store/todoReducer.tsx [hunk 3]
```diff
@@ -10,16 +10,22 @@ import {
...
   isLogedin: false,
+  userName: "",
   categories: [],
   status: [],
 };
 
+// token
+// user
+// isLoggedIn
+
```
Comment (akash-dhande-ib) [inline src/store/todoReducer.tsx:28]: I guess it doesn't belong here.

**Review by akash-dhande-ib — DISMISSED** [review]

**Review by akash-dhande-ib — APPROVED** [review]

**Pull request merged**